README.rst

```diff
@@ -86,40 +88,81 @@ request
 Creates an data export job request and return the export request id. To create a
 data export requests for all available tables for a course::
 
+    courseraresearchexports jobs request tables --course_id $COURSE_ID \
+        --purpose "testing data export"
+
+In order to know your course_id, you can take advantage
+of our COURSE API, putting in the appropriate `courseSlug`. 
```

Comment:
Here and below, courseSlug should be snake case. We can also remove the code styling since it isn't used for course_id.